### PR TITLE
add runner target updates

### PR DESCRIPTION
**Pull request by agentiqd**
might've went overboard on runner changes

### CODE REVIEW

File: .github/workflows/docker-build-deploy.yaml [hunk 1]
```diff
@@ -282,16 +282,17 @@ jobs:
           fi
         
   canary-deployment:
-    runs-on: [ ubuntu-20.04 ]
+    runs-on: [ self-hosted, linux, x64 ]
```
Comment (agentiqd) [inline .github/workflows/docker-build-deploy.yaml:285]: we might no _need_ to move this one, but I think keeping deploy internal is a good idea anyways

File: .github/workflows/slack-update.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ on:
       
 jobs:
   slack_notify:
-    runs-on: [ ubuntu-20.04 ]
+    runs-on: [ self-hosted, linux, x64 ]
```
Comment (agentiqd) [inline .github/workflows/slack-update.yaml:32]: my understanding this is the job that has the waiter/long running action, right? @BlaiseAgentIQ

Comment (BlaiseAgentIQ) [reply .github/workflows/slack-update.yaml:32]: That is correct

File: .github/workflows/docker-build-deploy.yaml [hunk 3]
```diff
@@ -66,7 +66,7 @@ jobs:
           echo "matrix=${ENVIRONMENT_IN}" >> $GITHUB_OUTPUT
 
   build_push:
-    runs-on: [ hub-runner ]
+    runs-on: [ self-hosted, linux, x64 ]
```
Comment (agentiqd) [inline .github/workflows/docker-build-deploy.yaml:69]: so this was still using the one-off "hub-runner"... kinda surprised that's still a valid runner

**Review by BlaiseAgentIQ — APPROVED** [review]

**Pull request merged**